template/module/tests/test_abstract_something.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Copyright <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import mock
+
+from openerp import models
+from openerp.tests.common import TransactionCase
+
+
+class AbstractSomethingTester(models.Model):
+    """ It provides a real model object to test the abstract with """
+    _name = 'abstract.something.tester'
+    _description = 'Abstract Something Tester'
+    _inherit = 'abstract.something'
+
+
+class TestAbstractSomething(TransactionCase):
+
+    @classmethod
+    def _init_test_model(cls, model_cls):
+        """ It builds a model from model_cls in order to test abstract models
+
+        Note that this does not actually create a table in the database, so
+        there may be some unidentified edge cases.
+
+        Args:
+            model_cls: (openerp.models.BaseModel) Class of model to initialize
```

Comment:
Note from https://github.com/OCA/server-tools/pull/681/files/aebfc5578c5e8d4fdca5065465d6066d588904ba#r98303294

There are some issues with this docblock:

- As currently written, the method does actually create a table for the model. That happens as part of `_auto_init()`
- The var name in the `Returns` section is wrong